rust/hyperlane-base/src/contract_sync/cursors/sequence_aware/backward.rs

```diff
@@ -9,10 +9,13 @@ use hyperlane_core::{
     HyperlaneSequenceAwareIndexerStoreReader, IndexMode, Indexed, LogMeta, SequenceIndexed,
 };
 use itertools::Itertools;
+use tokio::time::sleep;
 use tracing::{debug, instrument, warn};
 
 use super::{LastIndexedSnapshot, TargetSnapshot};
 
+const MAX_BACWARD_SYNC_BLOCKING_TIME: Duration = Duration::from_secs(1);
```

Comment:
nit: BACWARD -> BACKWARD

Wdyt about like 5 seconds? Because we're doing tokio::select! and I'm not sure how long one iteration of the skip_indexed loop is on the scraper, a little worried that 1 second could be too short to go through one iteration and we'd never make progress on the backward

Also wondering if this means we need to be careful about skip_indexed being cancel safe, maybe a comment there would be good

Reply:
> 5 seconds

was torn between 1 and 5 secs but thought 1 is more than enough for a couple of round trips to the db. Happy to go with 5

> cancellation safety

should be all good since we're cancelling a future rather than a task, which can only happen at an `await` point (https://blog.yoshuawuyts.com/async-cancellation-1/), so worst case we make the same `get_sequence_log_block_number` again when we resume